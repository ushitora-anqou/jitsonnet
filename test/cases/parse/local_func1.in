local a() = 0; a()
