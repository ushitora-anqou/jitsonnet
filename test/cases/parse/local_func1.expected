{ Syntax.v =
  (Syntax.Local
     ([(Syntax.BindFunc
          ("a",
           { Syntax.v =
             ([],
              { Syntax.v = (Syntax.Number 0.);
                loc =
                { Syntax.fname = "test/cases/parse/local_func1.in";
                  ran =
                  (Some { Syntax.startpos = { Syntax.line = 1; column = 12 };
                          endpos = { Syntax.line = 1; column = 13 } })
                  }
                });
             loc =
             { Syntax.fname = "test/cases/parse/local_func1.in";
               ran =
               (Some { Syntax.startpos = { Syntax.line = 1; column = 6 };
                       endpos = { Syntax.line = 1; column = 13 } })
               }
             }))
        ],
      { Syntax.v =
        (Syntax.Call
           ({ Syntax.v = (Syntax.Var "a");
              loc =
              { Syntax.fname = "test/cases/parse/local_func1.in";
                ran =
                (Some { Syntax.startpos = { Syntax.line = 1; column = 15 };
                        endpos = { Syntax.line = 1; column = 16 } })
                }
              },
            ([], []), false));
        loc =
        { Syntax.fname = "test/cases/parse/local_func1.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 15 };
                  endpos = { Syntax.line = 1; column = 18 } })
          }
        }));
  loc =
  { Syntax.fname = "test/cases/parse/local_func1.in";
    ran =
    (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
            endpos = { Syntax.line = 1; column = 18 } })
    }
  }